### DISCUSSION TITLE
Grid Path on AWS

### DISCUSSION THREAD

**Discussion by ekgold42** [blue-marble/gridpath#1136] [Q&A] [answered]
Has anyone run GridPath on AWS instances? We have been running tests locally, but we will need more compute for the models we eventually want to run. We unfortunately have no experience with AWS yet, but it seems like the obvious path. Any advice, expected difficulties, or reasons this is a bad idea and suggestions for better ideas would be much appreciated.

Thank you!

**Comment by anamileva** [comment, marked as answer]
I frequently run GridPath on Google Compute Engine instances and am aware of others who run it on AWS as well as university and national lab high-performance computing. You can install GridPath on a remote machine just like you do locally. Depending on the problem(s) you are solving and your cloud computing budget, it may make sense to use one large instance or many small ones. You can reach out to support@gridpath.io if you need help.

**Reply by ekgold42** [reply]
Thank you, if we get stuck, we may reach out!
